automation/settings.ps1

```diff
@@ -0,0 +1,192 @@
+#  CURSEFORGE PROJECT SETTINGS
+# =====================================================================//
+
+# Modpack's Minecraft Version
+# @(6756) - is Minecraft 1.12.2
+# @(8203) - is Minecraft 1.16.5
+# More can be found by running GetGameVersions
+$GAME_VERSIONS = @(9008)
+
+# Can be "alpha", "beta" or "release"
+$CLIENT_RELEASE_TYPE = "release"
+
+#=====================================================================//
+#  DEPENDENCIES
+#=====================================================================//
+
+# File name of the latest https://github.com/ModdingX/ModListCreator/releases
+$CHANGELOG_GENERATOR_JAR = "ModListCreator-4.0.2-fatjar.jar"
+
+# File name of the latest https://github.com/ModdingX/ModListCreator/releases
+$MODLIST_CREATOR_JAR = "ModListCreator-4.0.2-fatjar.jar"
+
+#=====================================================================//
+#  CLIENT FILE SETTINGS
+#=====================================================================//
+
+$CLIENT_FILE_AUTHOR = "itsDinkd"
+
+$FOLDERS_TO_INCLUDE_IN_CLIENT_FILES = @(
+    "bg",
+    "config",
+    "defaultconfigs",
+    "emotes",
+    "kubejs",
+    "paintings",
+    "shaderpacks",
+    "resourcepacks",
+    "servers.dat"
+)
+
+$CONFIGS_TO_REMOVE_FROM_CLIENT_FILES = @("roughlyenoughitems")
+
+$FOLDERS_TO_REMOVE_FROM_CLIENT_FILES = @("local/ftbutilities")
+
+#=====================================================================//
+#  SERVER FILE SETTINGS
+#=====================================================================//
+
+$SERVER_FILES_FOLDER = "$INSTANCE_ROOT/server_files"
+
+$SERVER_SETUP_CONFIG_PATH = "$SERVER_FILES_FOLDER/server-setup-config.yaml"
+
+# =====================================================================//
+#  MODULES
+# =====================================================================//
+
+# Toggle automatic building of the manifest zip on/off
+# Default: $true
+$ENABLE_CLIENT_FILE_MODULE = $true
+
+# Toggle the modpack uploader on/off
+# Setting this to $false will also disable the Server File and Changelog Generator Modules.
+# Default: $true
+$ENABLE_MODPACK_UPLOADER_MODULE = $true
+
+# Toggle server file feature on/off
+# Default: $true
+$ENABLE_SERVER_FILE_MODULE = $true
+
+# Toggle serverstarter compatibility on/off
+# This will update the "modpackUrl" in the file found at $SERVER_SETUP_CONFIG_PATH
+# to point to your newly created client files on the CurseForge CDN.
+# Default: $false
+$ENABLE_SERVERSTARTER_MODULE = $true
+
+# Toggle automatic changelog generator on/off
+# This module requires an older modpack manifest zip to be present, 
+# $LAST_MODPACK_VERSION must be set, and the manifest naming must be consistent.
+# Default: $false
+$ENABLE_CHANGELOG_GENERATOR_MODULE = $true
+# Path to the ChangelogGenerator's output file
+$CHANGELOG_PATH = "$INSTANCE_ROOT/changelogs/changelog_mods_$MODPACK_VERSION.md"
+
+# Toggle creation of a modlist file on/off
+# Default: $true
+$ENABLE_MODLIST_CREATOR_MODULE = $true
```

Comment:
modlist generator is enabled, see: https://github.com/itsdinkd/AQM2/blob/23ffefab59b7cbe3489e2ef7503c2d31f28d07be/changelogs/modlist_1.5.1.md

it generates a top level MODLIST.md and a version specific `changelogs/modlist_VER.md` file